csmgr/csmgr.py

```diff
@@ -211,6 +213,43 @@ async def addcreator(
 
         await safe_add_role(ctx, member, self.cog_creator_role)
         await ctx.send(f"Done. {member.mention} is now a cog creator!")
+    
+    @is_core_dev_or_qa()
+    @commands.command()
+    async def removecreator(self, ctx: commands.Context, user: Union[discord.Member, int]) -> None:
+        """
+        Unregister a cog creator
+        """
+        if not isinstance(user, discord.Member):
+            user_id = user
+            user = ctx.bot.get_user(user_id)
+            if not user:
+                try:
+                    user = await ctx.bot.fetch_user(user_id)
+                except discord.NotFound:
+                    return await ctx.send("User not found for that user id.")
+                except discord.HTTPException:
+                    return await ctx.send("Something went wrong while fetching the user.")
```

Comment:
This whole sections is unneeded - `user` is only really needed if they're member of the server and in that case it will be of type `discord.Member`